lib/DDG/Spice/Github.pm

```diff
@@ -16,10 +16,10 @@ triggers startend => "github";
 spice to => 'https://api.github.com/legacy/repos/search/$1?callback={{callback}}';
 spice proxy_cache_valid => '200 30d';
 
-handle query_lc => sub {
-    s/^github\s+|\s+github$//;
-    return $_ unless /^jobs\b|\bjobs$|^status\b|\bstatus$/;
-    return;
+handle remainder_lc => sub {
+	return unless $_;
+	return $_ unless /^jobs\b|\bjobs$|^status\b|\bstatus$/;
+	return;
```

Comment:
Looks like you're still indenting with *TAB*, you should be able to switch your editor to indent using spaces when you hit the TAB key or just hit the spacebar 4 times :smiley: